pkg/prop/prop.go

```diff
@@ -229,16 +229,18 @@ func (c *comparisons) fitnessDistance() (float64, bool) {
 
 // VideoConstraints represents a video's constraints
 type VideoConstraints struct {
-	Width, Height IntConstraint
-	FrameRate     FloatConstraint
-	FrameFormat   FrameFormatConstraint
+	Width, Height  IntConstraint
+	FrameRate      FloatConstraint
+	FrameFormat    FrameFormatConstraint
+	ReadTimeoutSec IntConstraint
```

Comment:
As it is a driver specific config rather than a device selection constraint, I think it should be only in `pkg/driver/camera/`.
Possible implementation would be
- add package global variable to be configured (simple but might cause problem when set by multiple places)
- add package global unexported variable and set by compile option (by `-ldflags '-X module.import.path.variable=value'`)

Reply:
another possible implementation is:
- get value from an environment variable

Reply:
I implemented a function in `camera_linux` to read the value from the `CAMERA_READ_TIMEOUT` environment variable if it exists and is appropriate.